### ISSUE TITLE
Can't customize push notification UI

### ISSUE THREAD

**Issue by itimmisriis** [chat-sdk/chat-sdk-android#420]
1. **Is the bug present in the demo Chat SDK project?**
Yes

2. **What modifications have you made to the Chat SDK?**
I have tried to extend different classes from ChatSDK but it seems impossible to change push notification without importing the entire source code into our project just to update the NotificationUtils class. 

This is less than ideal, as it becomes very difficult to keep our ChatSDK updated without having to update the source code on every update. It also makes our projects codebase much bigger than needed.

3. **Android Version:**
7.0, 8.0

4. **Steps taken to reproduce the problem:**
Can not customize UI for push notifications

5. **Expected result:**
Should be able to customize UI for push notifications without pulling in entire code base. Should be able to do natively through the SDK.

6. **Actual result:**
Entire codebase needs to be pulled in to make small customizations. 

7. **Comments:**
This would be extremely useful fix for everyone.

**Comment by bensmiley** [comment]
This is fixed in `4.3.8`. You can override the following method in the interface adapter:

```
    public NotificationDisplayHandler notificationDisplayHandler () {
        if(notificationDisplayHandler == null) {
            notificationDisplayHandler = new NotificationDisplayHandler();
        }
        return notificationDisplayHandler;
    }
```

**Comment by itimmisriis** [comment]
@bensmiley 

When will 4.3.8 be available? It looks like 4.3.7 is the newest version on the master branch.

**Comment by bensmiley** [comment]
It’s already on bintray. Only the Guthub docs haven’t been updated yet.
